src/client/ContentManagement.js

```diff
@@ -1006,7 +1006,8 @@ exports.PublishContentVersion = async function({objectId, versionHash, awaitComm
   });
 
   const abi = await this.ContractAbi({id: objectId});
-  const fromBlock = commit.blockNumber + 1;
+  const fromBlock = commit.blockNumber - 10; // due to block re-org
```

Comment:
We would like to make this `30` to cover the worst case observed (25 sec and 20 blocks).

Reply:
made the changes